assets/wizards/popups/views/segmentation/SingleSegment.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * WordPress dependencies.
+ */
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { find } from 'lodash';
+
+/**
+ * Internal dependencies.
+ */
+import { Button, TextControl, CheckboxControl, Router } from '../../../../components/src';
+
+const { NavLink, useHistory } = Router;
```

Comment:
After running `npm ci && npm run build`, I keep getting a `TypeError: useHistory is not a function` error that prevents me from viewing the Campaigns wizard screens.

Reply:
Can you also try resetting the npm packages in `assets/components`?

Reply:
That fixes the `TypeError`, but I still get the `format` error:

```
Uncaught (in promise) TypeError: Cannot read property 'format' of undefined

SegmentsList.js:72
```

Reply:
d52a2cc0388f0ecc0a418b6c4b054a67701e1003 should fix it, let me know!